src/commands/Events.ts

```diff
@@ -83,10 +85,33 @@ async function handleEventCreation(
       // Retrieve the discord event manager.
       const guildScheduledEventManger = interaction.guild?.scheduledEvents;
       // Create an event with the given options.
-      await guildScheduledEventManger?.create(eventOptions);
-
-      const content = `\`${eventName}\` event scheduled!`;
-      await interaction.followUp({ content });
+      try {
+        const discordEvent = await guildScheduledEventManger?.create(eventOptions);
+        const eventLink = discordEvent?.url;
+
+        const content = `\`${eventName}\` event scheduled!`;
+
+        await interaction.followUp({
+          ephemeral: true,
+          content,
+          fetchReply: true,
```

Comment:
@vongarcia97 we don't need to fetch the reply in this case. reason being that we're not acting on this message after its created.